ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -46,20 +49,39 @@ const UPLOAD_PEDIGREE_FIELD = {
 }
 
 const AGREE_CHECKBOX = {
-  name: 'agreeCheckbox',
+  name: 'agreeSeqrAccess',
   component: BooleanCheckbox,
   label: 'By submitting this form I agree to grant seqr access to the data in the associated workspace',
 }
 
 const FORM_FIELDS = [PROJECT_DESC_FIELD, UPLOAD_PEDIGREE_FIELD, GENOME_VERSION_FIELD, AGREE_CHECKBOX]
 
+const createProjectFromWorkspace = (values, namespace, name) => {
+  if (values[FILE_FIELD_NAME]) {
+    values.uploadedFileId = values[FILE_FIELD_NAME].uploadedFileId
+    delete values[FILE_FIELD_NAME]
```

Comment:
this is unnecessary. In the api, just check for the correct nested json instead of a top level `uploadedFileId` key

Reply:
There are quite some unrelated data in `values[FILE_FIELD_NAME]`. Is it okay to send them to the backend?

Reply:
yeah should be fine, we can always ignore them